samples/star-wars-api/HttpHandlers.fs

```diff
@@ -55,53 +56,100 @@ module HttpHandlers =
-                                        logger.LogTrace($"GraphQL subscription data:{Environment.NewLine}{{data}}", (JsonSerializer.Serialize(data, jsonSerializerOptions)))
-                                | SubscriptionErrors (null, errors) ->
-                                    logger.LogInformation("Produced GraphQL subscription errors")
-                                    if logger.IsEnabled LogLevel.Trace then
-                                        logger.LogTrace($"GraphQL subscription errors:{Environment.NewLine}{{errors}}", errors)
-                                | SubscriptionErrors (data, errors) ->
-                                    logger.LogInformation("Produced GraphQL subscription result with errors")
-                                    if logger.IsEnabled LogLevel.Trace then
-                                        logger.LogTrace($"GraphQL subscription errors:{Environment.NewLine}{{errors}}{Environment.NewLine}GraphQL deferred data:{Environment.NewLine}{{data}}", errors, (JsonSerializer.Serialize(data, jsonSerializerOptions)))
-                            )
+                        stream
+                        |> Observable.add (function
+                            | SubscriptionResult data ->
+                                logger.LogInformation ("Produced GraphQL subscription result")
+
+                                if logger.IsEnabled LogLevel.Trace then
+                                    logger.LogTrace (
+                                        $"GraphQL subscription data:{Environment.NewLine}{{data}}",
+                                        (JsonSerializer.Serialize (data, jsonSerializerOptions))
+                                    )
+                            | SubscriptionErrors (null, errors) ->
+                                logger.LogInformation ("Produced GraphQL subscription errors")
+
+                                if logger.IsEnabled LogLevel.Trace then
+                                    logger.LogTrace ($"GraphQL subscription errors:{Environment.NewLine}{{errors}}", errors)
+                            | SubscriptionErrors (data, errors) ->
+                                logger.LogInformation ("Produced GraphQL subscription result with errors")
+
+                                if logger.IsEnabled LogLevel.Trace then
+                                    logger.LogTrace (
+                                        $"GraphQL subscription errors:{Environment.NewLine}{{errors}}{Environment.NewLine}GraphQL deferred data:{Environment.NewLine}{{data}}",
+                                        errors,
+                                        (JsonSerializer.Serialize (data, jsonSerializerOptions))
+                                    ))
+
                     GQLResponse.Stream documentId
 
-                | RequestError errs ->
-                    GQLResponse.RequestError (documentId, errs)
+                | RequestError errs -> GQLResponse.RequestError (documentId, errs)
```

Comment:
Do we have logging in this case in another branch?

Reply:
Added a logging string for this.